ushahidi/platform-client-mzima
# Fix delay in UI while permissions are loading

This PR adds a spinner the permissions-list when editing or creating a role. 
How to test:
- Logged in as a user with role-creating permissions
- Go to settings, click on roles
- Click on a role or create a new one
- [ ] Spinner appears while the permissions are fetched from the API

Fix for issue https://linear.app/ushahidi/issue/USH-856/role-fields-appear-empty-for-a-sec-and-then-populated-with-data#comment-6cd75721
